samples/modules/lvgl/demos/prj.conf

```diff
@@ -3,7 +3,7 @@ CONFIG_LOG=y
 CONFIG_SHELL=y
 
 CONFIG_LVGL=y
-CONFIG_LV_Z_MEM_POOL_NUMBER_BLOCKS=8
+CONFIG_LV_Z_MEM_POOL_SIZE=16384
```

Comment:
Please squash in one commit, otherwise not bisectable.

Reply:
Sure thing.